pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -387,6 +415,11 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 			if mir.Location == "" {
 				return &InvalidRegistries{s: "invalid condition: mirror location is unset"}
 			}
+
+			if mir.DigestOnly && mir.TagOnly {
+				msg := fmt.Sprintf("invalid condition: set digest-only and tag-only for the same mirror %s", mir.Location)
```

Comment:
I’m not sure the `invalid condition: ` part is useful here. (I realize the error above starts the same way.)

Probably “digest-only and tag-only (is) set on the same mirror”, I tend to interpret “set …” as a command.